src/XRP/xrp-client-interface.ts

```diff
@@ -100,4 +100,17 @@ export default interface XrpClientInterface {
    *          and the final status of the transaction..
    */
   enableDepositAuth(wallet: Wallet): Promise<TransactionResult>
+
+  /**
+   * Enables DepositPreauth and authorizes an XRPL account to send to this XRPL account.
```

Comment:
I feel like there were some terminology changes made to the documentation on the base implementation of this - can you double check that things are the same here and in `DefaultXrpClient` or `XrpClientDecorator` or wherever else it is?

Reply:
there totally were, brb